src/lib.rs

```diff
@@ -11,9 +11,9 @@
 //! (https://github.com/nabijaczleweli/cargo-update/commit/f57e9c3#diff-a7b0a2dee0126cddf994326e705a91ea),
 //! and now either `windres`-`ar` combo or `RC.EXE` would be used, which was OK.
 //!
-//! Later it transpired, that [MSVC is even more incompatible with everything else]
+//! Later it transpired, that [Rust supports MSVC when not in a VC++ dev environment]
 //! (https://github.com/nabijaczleweli/cargo-update/commit/39fa758#diff-a7b0a2dee0126cddf994326e705a91ea)
-//! by way of not having `RC.EXE` in `$PATH` (because it would only be reasonable to do so),
+//! which means that `RC.EXE` is not in `$PATH` (because that would require a VC++ dev environment),
```

Comment:
Don't change Background, please. I's written in a tone that it's supposed to keep.